lib/api_scraper.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const axios = require("axios");
+
+class ApiScraper {
+  constructor(url) {
+    this.url = url;
+    this.user_agent =
+      "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.132 Safari/537.36";
+    this.headers = {
+      headers: {
+        "User-Agent": this.user_agent,
+      },
+    };
```

Comment:
How about using
```javascript
this.config = {
    headers: {
       "etc"
    }
}
```
this instead, it's less confusing

Reply:
I'm aware this change breaks things (spoiler, it's not going to be the last one), which is good, it forces us to do 2 things:

1. We force a consistent coding style
2. More unit tests!